src/components/graphs/bargraph/BarGraph.js

```diff
@@ -20,23 +79,50 @@ const def = {
 
 const BarGraph = ({ count }) => {
   const [barData, setBarData] = useState(def);
-  const [incidentCount, setIncidentCount] = useState(null);
+  const [horizBarData, setHorizBarData] = useState(def);
+  const [horizShowing, setHorizShowing] = useState(false);
 
   useEffect(() => {
-    setIncidentCount(count);
-  }, [count]);
+    const queryMatcher = window.matchMedia('(max-width: 1100px)');
+    setHorizShowing(queryMatcher.matches);
+    const callback = ({ matches }) => {
+      setHorizShowing(matches);
+    };
+    queryMatcher.addListener(callback);
+
+    return () => {
+      queryMatcher.removeListener(callback);
+    };
+  }, []);
 
   useEffect(() => {
-    if (incidentCount) {
```

Comment:
Do we know what happens if incidentCount is null? Seems like this was checking for that prior to creating the data object / dataset array

Reply:
It only checks if count is null, but considering we already have a default data set, this case is already covered, so I removed it.

Reply:
Ahhh ok, got it

Reply:
Actually, after testing it looks like GraphContainer is taking care of the case of a null count.